packages/jaeger-ui/vite.config.ts

```diff
@@ -83,7 +83,7 @@ export default defineConfig({
           '@layout-body-background': '#fff',
           '@layout-header-background': '#404040',
           '@layout-footer-background': '@layout-body-background',
-          '@layout-header-height': '64px',
+          '@layout-header-height': '48px',
```

Comment:
The Header's height was increased with 4.x upgrade. The resulting NavBar wasn't the one we had earlier. To revert this internal change, the height has to be decreased here.